i18n/en.json

```diff
@@ -0,0 +1,241 @@
+        "header": "{{imageBasename}} was successfully flashed {{targets}}"
+      },
+      "compatibility": {
+        "sizeNotRecommended": "Not recommended",
+        "tooSmall": "Too small",
+        "locked": "Locked",
+        "system": "System drive",
+        "containsImage": "Source drive",
+        "largeDrive": "Large drive"
+      },
+      "warning": {
+        "tooSmall": "The selected source is {{bytes}} larger than this drive.",
+        "exitWhileFlashing": [
+          "You are currently flashing a drive.",
+          "Closing Abitikku may leave your drive in an unusable state."
+        ],
+        "looksLikeWindowsImage": [
+          "It looks like you are trying to burn a Windows image.\n\n",
+          "Unlike other images, Windows images require special processing to be made bootable.",
+          "We suggest you use a tool specially designed for this purpose, such as",
+          "<a href=\"https://rufus.akeo.ie\">Rufus</a> (Windows),",
+          "<a href=\"https://github.com/slacka/WoeUSB\">WoeUSB</a> (Linux),",
+          "or Boot Camp Assistant (macOS)."
+        ],
+        "missingPartitionTable": [
+          "It looks like this is not a bootable image.\n\n",
+          "The image does not appear to contain a partition table,",
+          "and might not be recognized or bootable by your device."
+        ],
+        "driveMissingPartitionTable": "It looks like this is not a bootable drive. The drive does not appear to contain a partition table, and might not be recognized or bootable by your device",
+        "largeDriveSize": "This is a large drive! Make sure it doesn't contain files that you want to keep.",
+        "systemDrive": "Selecting your system drive is dangerous and will erase your drive!",
+        "sourceDrive": "Contains the image you chose to flash"
+      },
+      "error": {
+        "notEnoughSpaceInDrive": [
+          "Not enough space on the drive.",
+          "Please insert a larger one and try again."
+        ],
+        "genericFlashError": "Something went wrong. If it is a compressed image, please check that the archive is not corrupted.\n{{message}}",
+        "validation": [
+          "The write has been completed successfully but Abitikku detected potential",
+          "corruption issues when reading the image back from the drive.",
+          "\n\nPlease consider writing the image to a different drive."
+        ],
+        "openSource": "Something went wrong while opening {{sourceName}}\n\nError: {{errorMessage}}",
+        "flashFailure": "Something went wrong while writing {{imageBasename}} to $t(describeTarget, {\"count\": {{targets.length}}, \"desc\": \"{{targets.0.description}}\", \"name\": \"{{targets.0.displayName}}\"}).",
+        "driveUnplugged": [
+          "Looks like Abitikku lost access to the drive.",
+          "Did it get unplugged accidentally?",
+          "\n\nSometimes this error is caused by faulty readers that don't provide stable access to the drive."
```

Comment:
Should there be a comma after `Sometimes`?
I would personally write it like this: `This error can be sometimes caused by...`